nfc.py

```diff
@@ -60,6 +62,33 @@ def find(cls):
 	logging.basicConfig(level=logging.DEBUG)
 	logger = logging.getLogger(__name__)
 
+	logger.info("Checking for dependencies...")
+	if "pyusb" not in {pkg.key for pkg in pkg_resources.working_set}:
+		logger.info("pyusb is not available.")
+
+		if (input("Seems you are missing the required library \'pyusb\'.\nWould you like to install it now? (y/N): ") == 'y'):
+			try:
+				print("> pip install pyusb")
+				cp = subprocess.run(["pip", "install", "pyusb"], stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
```

Comment:
How do you know whether to use pip, pip3, setuptools, yum, apt or some local Python installation (/opt/python/bin/pip)? Install globally or locally? Into venv maybe?

Do not make assumptions, just ask the user to install it. Users knows best what they want. (And if they don't, then it's not our fault ;-) ).

Reply:
Makes sense. Will do that.